### PR TITLE
Remove Fee Estimator

### PR DESCRIPTION
**Pull request by mderasse**

### CODE REVIEW

**Comment by mderasse** [comment]
Need PR #98

**Review by effectsToCause — APPROVED** [review]

File: src/rpc/client.cpp [hunk 1]
```diff
@@ -34,7 +34,6 @@ static const CRPCConvertParam vRPCConvertParams[] =
     { "sendtoaddress", 1, "amount" },
     { "sendtoaddress", 4, "subtractfeefromamount" },
     { "sendtoaddress", 5 , "replaceable" },
-    { "sendtoaddress", 6 , "conf_target" },
```
Comment (hd4r) [inline src/rpc/client.cpp:37]: why are we removing this?

File: src/rpc/util.h [hunk 2]
```diff
@@ -79,9 +79,6 @@ CTxDestination AddAndGetMultisigDestination(const int required, const std::vecto
 
 UniValue DescribeAddress(const CTxDestination& dest);
 
-//! Parse a confirm target option and raise an RPC error if it is invalid.
-unsigned int ParseConfirmTarget(const UniValue& value, unsigned int max_target);
```
Comment (hd4r) [inline src/rpc/util.h:83]: How is this related to the fee estimator?

**Review by Calvario — APPROVED** [review]
